### DISCUSSION TITLE
Change the outline color of focused element to one specified in component

### DISCUSSION THREAD

**Discussion by istir** [chakra-ui/chakra-ui#4719] [Q&A] [answered]
Many questions about **removing** the outline, but how about just changing the color? 

I'm creating an app that lets the user change the default color scheme ``colorScheme={`${colorScheme}.500`}`` where `colorScheme` is one of `red`, `yellow`, `orange`... 
It works fine, except for when I click on something and the same boring blue outline appears, which sometimes look very off: 
![image](https://user-images.githubusercontent.com/22329399/133442929-c9805df2-e191-40bc-a132-d9823df55a58.png)

Is there any way to easily set the color of it just like colorScheme, preverably on a component basis and using chakra colors (eg. red.200)?

**Comment by aaronkik** [comment, marked as answer]
Hi,

I don't know how your app is set up to change the colorScheme globally but you could try extending the components. 

I've created a sandbox of how I override the Button component to use chakra css variables for the focus color.

https://codesandbox.io/s/chakra-outline-with-colorscheme-lztq9?file=/src/index.js

**Reply by istir** [reply]
That is exactly what I needed! I tried doing something similar, but tried setting _focus to 
``boxShadow: `0 0 0 3px ${c}.200` ``
instead of 
``boxShadow: `0 0 0 3px var(--chakra-colors-${c}-200)` ``
